internal/analytics/client/async/client.go

```diff
@@ -51,6 +53,8 @@ func New(svcModel *model.SvcModel, cfg *config.Instance, auth *authentication.Au
 	a.output = o
 	a.projectNameSpace = projectNameSpace
 	a.auth = auth
+	a.ci = condition.OnCI()
+	a.interactive = out.Config().Interactive
```

Comment:
Determining interactive in the main state tool function is a bit more elaborate. Do we want to include the extra checks? See here: https://github.com/ActiveState/cli/blob/master/cmd/state/main.go#L101

Reply:
There is an open ticket to reconcile the differences: https://activestatef.atlassian.net/browse/DX-1667. This is possibly the third case now where we use `output`'s determination of interactive over `main`'s, so majority wins to me.